catalog/spatial/geocoding/spatial-geocoding-feature/src/main/java/org/codice/ddf/spatial/geocoding/feature/CatalogFeatureIndexer.java

```diff
@@ -125,7 +130,8 @@ private void createOrUpdateMetacardForFeature(SimpleFeature feature, boolean cre
   }
 
   private Metacard createMetacardForFeature(SimpleFeature feature) throws FeatureIndexingException {
-    Metacard metacard = new MetacardImpl(metacardType);
+    Validate.notEmpty(metacardType, GEO_ENTRY_METACARD_TYPE_MISSING_ERROR_MSG);
+    Metacard metacard = new MetacardImpl(metacardType.get(0));
```

Comment:
✏️ why the `0` what's magic about it :trollface: ?

Reply:
There should only ever be one.  I am using a reference list to avoid blueprint dampening.  I think this might be a new common pattern to avoid service references from taking down on service restart or blocking a bundle from starting.

I could change this to a stream `findFirst` to make it more clear.

Reply:
I think just creating a constant would make it more clear: `final int THE_ONE_METACARD_TO_RULE_THEM_ALL = 0;`